default/scripting/species/common/telepathic.macros

```diff
@@ -0,0 +1,16 @@
+TELEPATHIC_DETECTION
+'''     EffectsGroup
+            description = "TELEPATHIC_DETECTION"
+            scope = And [
+                PopulationCenter
+                Not OR [
+                    Species name = Source.Species
+                    Species name = "SP_EXOBOT"
```

Comment:
I think that's actually a piece of code that got carried over from the original Xenophobic macros, where this line made sure the xenophobic effect doesn't get applied to the xenophobic species itself (xenophobes freaking themselves out, so to speak).

Which means the line probably should be removed here - after all, why should telepaths not be able to detect other telepaths/other pops of the same species as themselves? Of course, we can add in a TELEPATHIC_IMMUNE tag for species that are supposed to be immune to telepathic detection (which makes sense for dumb automatons like exobots).

That would be my solution - comments?